apps/elekto/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: elekto-config
+data:
+  APP_NAME: k8s.elections
+  APP_ENV: production
+  APP_KEY: 
+  APP_DEBUG: "False"
+  APP_URL: "http://localhost"
+  APP_PORT: 8080
+  APP_HOST: localhost
+  APP_CONNECT: http
+  DB_CONNECTION: mysql {or postgresql}
+  DB_HOST: {CloudSQL address}
+  DB_PORT: {CloudSQL port}
+  DB_DATABASE: {CloudSQL database}
```

Comment:
There are dynamic and will be added to the secret.

Reply:
ok, works, I'll take them out of the configmap